web-ng/src/app/components/project-list/project-list.component.scss

```diff
@@ -68,5 +68,5 @@
 }
 
 .container-projects {
-  background-color: var(--project-list-background);
+  background-color: mat-color($offWhite);
```

Comment:
CSS vars usually use `slug-case`, not `lowerCamelCase`.

Reply:
removed.